### DISCUSSION TITLE
Welcome to cursor-vip Discussions!

### DISCUSSION THREAD

**Discussion by kingparks** [kingparks/cursor-vip#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by imtiazfahim256** [comment]
PayPal is not supported in our country. Can you add card payment as an option? Also, is there any other discussion platform like Discord or Telegram? If so, please provide the address.

**Reply by kingparks** [reply]
https://github.com/kingparks/cursor-vip/issues/55 support crypto;
There is currently no other place for discussion

**Reply by imtiazfahim256** [reply]
can you please provide me the price and  crypto link

**Comment by helanhao** [comment]
使用代理模式发现AI回复问题的速度明显变的很慢

**Comment by passbyyt** [comment]
Our servers are currently overloaded for non-pro users, and
you've used your free quota. Please try again in a few minutes. If
you think this is a mistake, please contact hi@cursor.sh

**Reply by kingparks** [reply]
> Our servers are currently overloaded for non-pro users, and you've used your free quota. Please try again in a few minutes. If you think this is a mistake, please contact [hi@cursor.sh](mailto:hi@cursor.sh)

New chat can fixed

**Comment by passbyyt** [comment]
您好：
     你发的邮件已收到。
                      谢谢

**Comment by Brill2410** [comment]


**Reply by kingparks** [reply]
发我订单号 和 设备码 我来处理下

**Reply by Brill2410** [reply]
订单号：2024121922001498761429934744
商家订单号：i5YsvPCK5HRTi3gGGremqH
设备码：9429b027695f4257b7e474b92c8dccbf

另外，我这里有很多人有需求，能加个微信聊聊吗

**Comment by dangweilinshinidie** [comment]
我昨天机器码到期了，就换了一个，这个设备码就是机器码吗？我看设备码没有变化，而且也不能自动打开cursor了，但是账号能登陆上，报错是图片上的
![Uploading 屏幕截图 2024-12-20 175504.png…]()

**Comment by dangweilinshinidie** [comment]
![屏幕截图 2024-12-20 175504](https://github.com/user-attachments/assets/ad8def88-0034-4a8e-a9e4-8718ecf3ed27)

**Reply by kingparks** [reply]
> ![屏幕截图 2024-12-20 175504](https://private-user-images.githubusercontent.com/188686989/397706226-ad8def88-0034-4a8e-a9e4-8718ecf3ed27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3NTIxNzcsIm5iZiI6MTczNDc1MTg3NywicGF0aCI6Ii8xODg2ODY5ODkvMzk3NzA2MjI2LWFkOGRlZjg4LTAwMzQtNGE4ZS1hOWU0LTg3MThlY2YzZWQyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyMVQwMzMxMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YjRhZGFjOGYyYmU1MDkwMmVhYTRjNWExNzVkMmE1OWE5NWZlNDUxMmFiNzI1NGVmZTgxNzg1MGRkNmFkZDBmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.x7D84sCWD7KUpv-uQoWoRWdUoUsjzNT8A9efkqIIxFg)

这个不是你那边的原因，是普通模式的问题，普通模式有时就是会有这种问题，可以改用代理模式

**Comment by Nicholas-Hollan** [comment]
<img width="1136" alt="截屏2025-02-10 13 22 46" src="https://github.com/user-attachments/assets/c8360339-0e59-4d72-8c2d-9602ae4887b4" />
我是用的是模式3，但是为什么对话有限制（使用的是200一年的）然后感觉好几次发送都生成思考很长时间（需要stop然后重新submit才成功）
![Uploading 截屏2025-02-10 13.24.33.png…]()
还有就是我想添加google的studio模型，但是这个verify的APIkey很长时间都不完成

**Comment by passbyyt** [comment]
您好：
     你发的邮件已收到。
                      谢谢

**Comment by insking123** [comment]
我已捐赠，但是按回车提示“未捐赠,请捐赠完成后回车”。
设备码:762ad3ecdf394f5fbb8cf589a23515d1
订单号：2025041222001429301459525047
商家订单号：4j5kNBf5nCLPwmgpUVBGNi
麻烦帮我处理一下，谢谢。

**Reply by kingparks** [reply]
> 我已捐赠，但是按回车提示“未捐赠,请捐赠完成后回车”。 设备码:762ad3ecdf394f5fbb8cf589a23515d1 订单号：2025041222001429301459525047 商家订单号：4j5kNBf5nCLPwmgpUVBGNi 麻烦帮我处理一下，谢谢。

收到

**Reply by kingparks** [reply]
> 我已捐赠，但是按回车提示“未捐赠,请捐赠完成后回车”。 设备码:762ad3ecdf394f5fbb8cf589a23515d1 订单号：2025041222001429301459525047 商家订单号：4j5kNBf5nCLPwmgpUVBGNi 麻烦帮我处理一下，谢谢。

处理好了 
2026-04-19 17:50:58 到期
感谢支持

**Reply by insking123** [reply]
cursor version：0.48.8
TID1: NGdoyMlU
TID2: NGdoyMlU
授权成功！使用过程请不要关闭此窗口
371d 19h 24m 37s  
is ok,TKU

**Reply by insking123** [reply]
出现：Our servers are currently overloaded for non-pro 说明sm3的 claude 的阶段性额度用完了，通过快捷键 u3d、u3t、u3h 手动刷新账号
为了防止恶意多次刷新账号，增加了小额支付验证
如果提示暂无新账号，只能等待或先用着v0.45版本
Switch to English：Press 'sen' on keyboard in turn
小额付费刷新账号：依次按键盘 u3d
10x小额付费刷新账号：依次按键盘 u3t
100x小额付费刷新账号：依次按键盘 u3h
订阅时长会在验证通过后增加对应的天数

200这种怎么刷新账号，试了好多次，就跳到
“捐赠完成后请依次按键 c3h
371d 18h 46m 17schecking...
371d 18h 46m 13s未捐赠,请捐赠完成后回车
371d 18h 43m 58schecking...
371d 18h 43m 54s未捐赠,请捐赠完成后回车” 
模型都用不了呢

**Comment by passbyyt** [comment]
您好：
     你发的邮件已收到。
                      谢谢

**Comment by insking123** [comment]
捐赠200没法刷新帐号呢，
输入u3d、u3t、u3h 都没用
然后弹出捐赠链接，提示捐赠完成后请依次按键 c3h
但是按c3h没响应。
未捐赠,请捐赠完成后回车

**Reply by kingparks** [reply]
> 捐赠200没法刷新帐号呢， 输入u3d、u3t、u3h 都没用 然后弹出捐赠链接，提示捐赠完成后请依次按键 c3h 但是按c3h没响应。 未捐赠,请捐赠完成后回车

小额刷新支付了吗？

**Comment by Joost11568798946** [comment]
我已捐赠，但是按回车提示“未捐赠,请捐赠完成后回车”。
设备码:7d4dcdf50bc247afba78d7f4109a9927
订单号：2025041622001472061455254412
商家订单号：edNrPg8TjqAyZ5Lh4uwMVE
麻烦帮我处理一下，谢谢。

**Reply by kingparks** [reply]
> 我已捐赠，但是按回车提示“未捐赠,请捐赠完成后回车”。 设备码:7d4dcdf50bc247afba78d7f4109a9927 订单号：2025041622001472061455254412 商家订单号：edNrPg8TjqAyZ5Lh4uwMVE 麻烦帮我处理一下，谢谢。

已核实并处理完成
2026-04-16 17:08:24 到期
感谢支持

**Comment by wutonghands** [comment]
我现在需要更换设备，原来的设备更换不用了，如何将当前的设备id迁移到新的设备上  
![微信截图_20250416184431](https://github.com/user-attachments/assets/ad7407e3-e352-42bf-a473-c698c21db9a5)

**Reply by kingparks** [reply]
> 我现在需要更换设备，原来的设备更换不用了，如何将当前的设备id迁移到新的设备上 ![微信截图_20250416184431](https://private-user-images.githubusercontent.com/89695151/434290591-ad7407e3-e352-42bf-a473-c698c21db9a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ4MTk0NDIsIm5iZiI6MTc0NDgxOTE0MiwicGF0aCI6Ii84OTY5NTE1MS80MzQyOTA1OTEtYWQ3NDA3ZTMtZTM1Mi00MmJmLWE0NzMtYzY5OGMyMWRiOWE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE2VDE1NTkwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwOWQ0NGE2NTBhM2NhODc1MmI1ZWY4ZjVkYmFiMmZiM2IzZjM2YTY4Mzc3YWZmZjlhN2E1NTU0YzEzOGNhOWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SeRpQtJ9Cb3lEFydxtsdmVhHizQPhL5lyKu2RIFLIEQ)

年付费用户免费迁移一次，发我新老设备码，老设备码找不到也可以发我商家订单号，我来做迁移

**Comment by passbyyt** [comment]
您好：
     你发的邮件已收到。
                      谢谢

**Comment by Anna8956** [comment]
use claude-4-connet:Visit cursor.com/settings and pay your invoice in Stripe to resume requests.

**Reply by Anna8956** [reply]
I want use claude 3.7 thinking

**Reply by nongh4704** [reply]
the same problem，how to fix it

**Comment by P47IENT0** [comment]
您好，我的设备做过一次迁移，老的设备码是0b0eca975494402b87377c0425239f11，新的设备码是：2e95eaa336aa4b3cad50fbec356c00bb

![image](https://github.com/user-attachments/assets/5158f61f-1361-4c0c-b5e0-f76daf0931a7)

**Reply by kingparks** [reply]
> 2e95eaa336aa4b3cad50fbec356c00bb

迁移好了

**Comment by Joost11568798946** [comment]
您好，重新装了一下Windows系统，然后设备码就变了，之前买的就不能用了，如何进行迁移？
旧设备码：7d4dcdf50bc247afba78d7f4109a9927
新设备码：8f44f9c4e3d04e3c8074babde7d00ea0

**Reply by kingparks** [reply]
> 您好，重新装了一下Windows系统，然后设备码就变了，之前买的就不能用了，如何进行迁移？ 旧设备码：7d4dcdf50bc247afba78d7f4109a9927 新设备码：8f44f9c4e3d04e3c8074babde7d00ea0

年付费用户可免费迁移一次，已迁移完成

**Comment by ZSYLOVE** [comment]
<img width="371" height="348" alt="{7A066854-A9B3-485B-B421-BCA6571186E6}" src="https://github.com/user-attachments/assets/7c1b9762-eaf7-436a-8fff-d6ebe7e2a483" /> 这种问题也要刷新吗？

**Comment by passbyyt** [comment]
您好：
     你发的邮件已收到。
                      谢谢

**Comment by wangqiaodan123** [comment]
为什么新建聊天还是这样
<img width="1070" height="105" alt="image" src="https://github.com/user-attachments/assets/f8300f9f-88c4-4cfa-b2d3-c4975420d6ba" />

**Comment by P47IENT0** [comment]
<img width="1713" height="234" alt="image" src="https://github.com/user-attachments/assets/fdb7f145-32c6-4cec-aa89-959639ea26de" />
您好，这两天使用cursor-vip好像遇到了网络问题报错，请问这个是哪部分出问题了

**Reply by kingparks** [reply]
> <img alt="image" width="1713" height="234" src="https://private-user-images.githubusercontent.com/147136978/470588020-fdb7f145-32c6-4cec-aa89-959639ea26de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM0MTgwMzEsIm5iZiI6MTc1MzQxNzczMSwicGF0aCI6Ii8xNDcxMzY5NzgvNDcwNTg4MDIwLWZkYjdmMTQ1LTMyYzYtNGNlYy1hYTg5LTk1OTYzOWVhMjZkZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNVQwNDI4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMzg0NjQ1MDgwNGM1YTQ3ZDdiNDc2ZGNjMDkyM2I1M2FmZjE2ODIwYjlmMGYyYTBkMDRmYmFkYTkyNGRlZGVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.lqec_s-GiAD74hk6BU6ix9ovPvfa841pkW81lGUKtOw"> 您好，这两天使用cursor-vip好像遇到了网络问题报错，请问这个是哪部分出问题了

网络的问题，可以先用手机热点启动cursor-vip后,在更换为常用的网络

**Comment by wangqiaodan123** [comment]
没用
<img width="1627" height="1139" alt="image" src="https://github.com/user-attachments/assets/60c3b575-b4f2-486f-b36d-6c6a1513a84c" />

**Reply by kingparks** [reply]
> 没用 <img alt="image" width="1627" height="1139" src="https://private-user-images.githubusercontent.com/185065323/470622767-60c3b575-b4f2-486f-b36d-6c6a1513a84c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM0MTgwMzEsIm5iZiI6MTc1MzQxNzczMSwicGF0aCI6Ii8xODUwNjUzMjMvNDcwNjIyNzY3LTYwYzNiNTc1LWI0ZjItNDg2Zi1iMzZkLTZjNmExNTEzYTg0Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNVQwNDI4NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OGVhYWQ0NzM3ZmU3MDdjN2I1NzViNmNjYjJmNmJhODIzZWY2ZjUzZWUxYjU2NmEyNjc2MGY5MjU0ZDRkOTE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.0VvOhI0gAOhENAENSRAaThC4RXn9bo4DXPXbm4Qw_5E">

u3d u3t u3h 更换新账号
u3o 更换老账号

**Reply by wangqiaodan123** [reply]
切换老账号没用，切换新账号需要捐赠

**Reply by kingparks** [reply]
> 切换老账号没用，切换新账号需要捐赠

是的

**Comment by ZSYLOVE** [comment]
<img width="371" height="348" alt="470166328-7c1b9762-eaf7-436a-8fff-d6ebe7e2a483" src="https://github.com/user-attachments/assets/157648b9-5600-4a06-a706-c7640a29e8ec" />
都已经缴费200了
，为什么还要捐赠刷新，小额刷新只能用30分钟左右为什么呢？

**Reply by kingparks** [reply]
> <img alt="470166328-7c1b9762-eaf7-436a-8fff-d6ebe7e2a483" width="371" height="348" src="https://private-user-images.githubusercontent.com/158316522/473828942-157648b9-5600-4a06-a706-c7640a29e8ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQzNjAyMTEsIm5iZiI6MTc1NDM1OTkxMSwicGF0aCI6Ii8xNTgzMTY1MjIvNDczODI4OTQyLTE1NzY0OGI5LTU2MDAtNGEwNi1hNzA2LWM3NjQwYTI5ZThlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNVQwMjExNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NWJhZTVmZmE2ODE4MWU5NDU2NDAxNGJiMTg3ZDRlYjUyODQ1MDcwMzc5NTQ0NWQyZDYwY2NmMjQ2MzcwNGYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bKft-nQNnuLrNOV1nSJKxS7Phkf-8zDFtwAB6eXxW3k"> 都已经缴费200了 ，为什么还要捐赠刷新，小额刷新只能用30分钟左右为什么呢？

到期后付费是增加工具使用时间，小额刷新付费验证也会增加使用时间。

**Reply by ZSYLOVE** [reply]
> > <img alt="470166328-7c1b9762-eaf7-436a-8fff-d6ebe7e2a483" width="371" height="348" src="https://private-user-images.githubusercontent.com/158316522/473828942-157648b9-5600-4a06-a706-c7640a29e8ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQzNjAyMTEsIm5iZiI6MTc1NDM1OTkxMSwicGF0aCI6Ii8xNTgzMTY1MjIvNDczODI4OTQyLTE1NzY0OGI5LTU2MDAtNGEwNi1hNzA2LWM3NjQwYTI5ZThlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNVQwMjExNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NWJhZTVmZmE2ODE4MWU5NDU2NDAxNGJiMTg3ZDRlYjUyODQ1MDcwMzc5NTQ0NWQyZDYwY2NmMjQ2MzcwNGYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bKft-nQNnuLrNOV1nSJKxS7Phkf-8zDFtwAB6eXxW3k">都已经缴费200了 ，为什么还要捐赠刷新，小额刷新只能用30分钟左右为什么呢？
> 
> 到期后付费是增加工具使用时间，小额刷新付费验证也会增加使用时间。

到期指的是？我的似乎没有到期吧，我是今年五月中旬购买的
<img width="1458" height="614" alt="{101F1119-146F-42D5-8F57-253FBA5E9307}" src="https://github.com/user-attachments/assets/3045188b-5167-43ed-b5e0-b018a3cb6170" />

**Comment by kingparks** [comment]
最后一行的 283d 就是指还有283天软件到期，跟小额刷新是两回事
